packages/dx-react-scheduler-demos/src/demo-sources/tooltip/material-ui/custom.jsx

```diff
@@ -33,32 +33,32 @@ const style = theme => ({
   },
 });
 
-const Head = withStyles(style, { name: 'Head' })(({
-  children, appointment, classes, ...restProps
+const Header = withStyles(style, { name: 'Header' })(({
+  children, appointmentData, classes, ...restProps
 }) => (
-  <AppointmentTooltip.Head
+  <AppointmentTooltip.Header
     {...restProps}
-    appointment={appointment}
-    className={classes.head}
+    appointment={appointmentData}
```

Comment:
Is this line required? Looks very strange, because of the `AppointmentTooltip.Header` has `appointmentData` property.